src/OpenTelemetry.AutoInstrumentation/Instrumentation.cs

```diff
@@ -79,6 +85,26 @@ public static void Initialize()
             _pluginManager = new PluginManager(GeneralSettings.Value);
             _pluginManager.Initializing();
 
+#if NET6_0_OR_GREATER
+            var profilerEnabled = GeneralSettings.Value.ProfilerEnabled;
+            var (threadSamplingEnabled, threadSamplingInterval, allocationSamplingEnabled, maxMemorySamplesPerMinute, exportInterval, continuousProfilerExporter) = _pluginManager.GetFirstContinuousConfiguration();
```

Comment:
Perform these operation and below condition check only when profiling is enabled.

Reply:
Fixes, please check: d00db398cc3fd31fd9d3721fce96187840c058c2